test/setup/main.tf

```diff
@@ -26,9 +26,37 @@ module "project" {
   skip_gcloud_download = true
 
   activate_apis = [
+    "cloudkms.googleapis.com",
     "cloudresourcemanager.googleapis.com",
     "bigquery.googleapis.com",
     "bigquerystorage.googleapis.com",
     "serviceusage.googleapis.com"
   ]
 }
+
+#Can't delete this with Terraform.
+module "kms_keyring" {
+  source     = "terraform-google-modules/kms/google"
+  version    = "~> 1.2"
+  depends_on = [module.project] #waits for API config
+
+  project_id      = module.project.project_id
+  location        = "us"
+  keyring         = "ci-bigquery-keyring"
+  keys            = ["foo"]
+  prevent_destroy = "false"
+}
+
+module "initialize_encryption_account" {
+  #pinned to prevent bug with random cache count evaluation, see #82
+  source     = "terraform-google-modules/gcloud/google"
+  version    = "1.2.0"
+  depends_on = [module.project] #waits for API config
+
+  platform              = "linux"
+  additional_components = ["bq"]
+  #skip_download         = true
```

Comment:
lets enable this

Reply:
this is not supported in the version we pinned.

Reply:
@kpeder any blockers with using the latest version `~>2.0` as the bug only occurs with explicit depends_on which now you have removed in https://github.com/terraform-google-modules/terraform-google-bigquery/pull/92/commits/01c6e0ba3344ec0958406dbbaec679f8a1bccd3d

Reply:
Ack, didn't realize they were directly related... sorry for the delay, had to wait out a Billing API issue to run project factory for tests.